client/src/components/BoardGameTile.js

```diff
@@ -5,7 +5,7 @@ const BoardGameTile = props => {
     const { id, name, rating } = props
     return(
         <div>
-            <li><Link to={`/boardgames/${id}`}>{name}</Link></li>
+            <li><Link to={`/board-games/${id}`}>{name}</Link></li>
```

Comment:
doesn't look like the Express clientRouter knows about this show route for board games (it has just "/board-games"